tests/foreman/api/test_subscription.py

```diff
@@ -443,30 +443,115 @@ def test_positive_expired_SCA_cert_handling(module_sca_manifest_org, rhel_conten
     assert rhel_contenthost.subscribed
 
 
-@pytest.mark.stubbed
 @pytest.mark.tier2
-def test_positive_os_restriction_on_repos():
-    """Verify that you can specify OS restrictions on custom repos
+@pytest.mark.rhel_ver_match('N-2')
```

Comment:
@damoore044 Since initially we didn't cherrypicked any of the EL10 related bits to 6.16.z or 6.15.z, so not sure if this logic is expected to work here?